### PR TITLE
Toggle zone test

### PR DESCRIPTION
**Pull request by mr-redFox**
added test for <ToggleZone />

### CODE REVIEW

**Review by skiffua — DISMISSED** [review]

File: src/client/components/toggleZone/toggleZone.test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import toJson from 'enzyme-to-json';
+
+import ToggleZone from './toggleZone';
+
+Enzyme.configure({ adapter: new Adapter() });
```
Comment (GeFeSTTS) [inline src/client/components/toggleZone/toggleZone.test.js]: Delete this string. Enzyme config already configured in settings.

File: src/client/components/toggleZone/toggleZone.test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  const fakeChilds = {
+    children: React.createElement('div'),
+  };
+
+  beforeEach(() => {
+    wrapper = shallow(<ToggleZone {...fakeChilds} />);
+  });
+
+  it('should render ToggleZone component properly', () => {
+    expect(toJson(wrapper)).toMatchSnapshot();
```
Comment (tonypntsrn) [inline src/client/components/toggleZone/toggleZone.test.js]: Do you actually need this toJson method here?

File: src/client/components/toggleZone/toggleZone.test.js (hunk 1 above)
Comment (mr-redFox) [reply src/client/components/toggleZone/toggleZone.test.js]: done

File: src/client/components/toggleZone/toggleZone.test.js (hunk 2 above)
Comment (mr-redFox) [reply src/client/components/toggleZone/toggleZone.test.js]: done

**Review by GeFeSTTS — APPROVED** [review]

**Review by skiffua — APPROVED** [review]

**Review by tonypntsrn — APPROVED** [review]

**Pull request merged**